config-windows.md

```diff
@@ -3,6 +3,21 @@
 This document describes the schema for the [Windows-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
 The Windows container specification uses APIs provided by the Windows Host Compute Service (HCS) to fulfill the spec.
 
+## <a name="configWindowsLayerFolders" />LayerFolders
+
+**`layerfolders`** (array of strings, REQUIRED) specifies a list of layer folders the container image relies on. The list is ordered from topmost layer to base layer.
```

Comment:
For required arrays, you probably also want to require at least one entry (otherwise you might as well make the property OPTIONAL).  See #769 for an example of the spec wording and `minItems` JSON Schema I've used for that sort of thing before.